### PR TITLE
fix: tooltip position when using bottom position

### PR DESCRIPTION
**Pull request by anttio**
## 📎 Jira ticket
https://transferwise.atlassian.net/browse/DS-240

## ❓ Context  
Using tooltip position bottom positioned the tooltip wrongly.

## 🚀 Changes 
Calculate the position of the tooltip based on the parent element height.

## ✅ Checklist

- [ ] All changes are covered by tests
- [ ] All changes have been crossbrowser checked, especially IE11
- [ ] The changes are covered in docs

### CODE REVIEW

File: packages/components/src/tooltip/Tooltip.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ function getTooltipStyle(parent, tooltip, position, offset) {
       };
     case Position.BOTTOM:
       return {
-        top: `${parent.offsetTop + tooltip.offsetHeight + offset}px`,
+        top: `${parent.offsetTop + parent.offsetHeight + offset}px`,
```
Comment (andreapiras) [inline packages/components/src/tooltip/Tooltip.js:26]: Is there any test covering this?

Comment (anttio) [reply packages/components/src/tooltip/Tooltip.js:26]: No. I think the tests on this component has never been that thorough.

Comment (andreapiras) [reply packages/components/src/tooltip/Tooltip.js:26]: We need to give some love to this component

Comment (helendurranttw) [reply packages/components/src/tooltip/Tooltip.js:26]: @andreapiras we have a ticket for that :)

**Pull request merged**